checkstyle/import-control.xml

```diff
@@ -330,6 +330,7 @@
         <allow pkg="kafka.zk" />
         <allow pkg="org.apache.kafka.tools"/>
         <allow pkg="org.apache.kafka.server.config" />
+        <allow pkg="org.apache.kafka.metadata.authorizer"/>
```

Comment:
Is this necessary? I remove it and build still works well.

Reply:
This not required.

Sorry for that, should been check this before review.